lib/timely.rb

```diff
@@ -5,6 +5,8 @@ module Timely
   require 'timely/string'
   require 'timely/date'
   require 'timely/time'
+  require 'timely/time_field'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.